HowTo topics for Kafka tiered storage (#2162)
aiven/devportal: docs/products/kafka/howto/tiered-storage-overview.rst

Access tiered storage overview
--------------------------------

1. In the Aiven Console, choose your project and select your Aiven for Apache Kafka service.
2. From the left sidebar, select **Tiered Storage**. This action will display an overview of tiered storage and its associated details.

Key insights of tiered storage
------------------------------

Get a quick snapshot of the essential metrics and details related to tiered storage:

- **Current billing expenses in USD**: Stay informed about your current tiered storage expenses.
- **Forecasted month cost in USD**: Estimate your upcoming monthly costs based on current usage.
- **Data tiered in bytes**: View the volume of data that has been tiered.
- **Storage overview**: Understand the specifics of the storage mediums in use, including details about the used object storage and SSD storage.

Current tiered storage configurations
---------------------------------------------

View of the current configurations for tiered storage:

- **Local Cache**: Shows the current cache configuration.

> - **Default Local Retention Time (ms)**: Shows the current local data retention set in milliseconds.

Reviewer: Wondering if these should mention the actual setting name? 🤔

Reviewer: Yes, those are the actual settings names. I'm just thinking that if for example you use the CLI, you would use the real setting names so it could be reassuring that we are in fact talking about the exact same service level settings. 

I think we should just use the real setting name in console as well to avoid confusion.

- **Default Local Retention Bytes**: Shows the configured volume of data, in bytes, for local retention.

To modify these settings:

1. In the **Tiered storage settings** section, select the ellipsis (three dots) and choose **Update tiered storage settings**.
2. Within **Update tiered storage settings** page, adjust the values for:
   
   - Local Cache
   - Default Local Retention Time (ms)
   - Default Local Retention Bytes
3. Confirm by selecting **Save changes**.

Graphical view of tiered storage costs
------------------------------------------

Gain a visual understanding of your tiered storage expenses:

- **Hourly expense**: Visualize your hourly expenses through graphical representation.

> - **Total Cost and forecast**: Get a clear picture of your overall costs and receive a forecast based on current trends.

Reviewer: It might be good idea to clarify what the forecast is based on and that it's just an guesstimate based on your latest usage.

Detailed storage overview
-------------------------

Explore the specifics of your storage usage and configurations:

> - **Used object storage and SSD storage**: Dive deep into the storage mediums in use.

Reviewer: Is it "Dive deep" or "deep dive"?

Author: Grammatically not incorrect, as both are idiomatic expressions in English, but "Deep dive" is more commonly used. I will change that.  
Dive Deep: This is a verb phrase. 
Deep Dive: This is a noun phrase.

Access tiered storage overview
--------------------------------

1. In the `Aiven console <https://console.aiven.io/>`_, choose your project and select your Aiven for Apache Kafka service.
2. From the left sidebar, select **Tiered Storage**.

   - If you haven't enabled tiered storage for your service, you'll have the option to enable it.
   - If tiered storage is enabled but not configured for any topics, you have the option to set it up for topics directly. For more details, see :doc:`Enable and configure tiered storage for topics </docs/products/kafka/howto/configure-topic-tiered-storage>`. 

3. Once configured, you can view an overview of tiered storage and its associated details.

Key insights of tiered storage
------------------------------

Get a quick snapshot of the essential metrics and details related to tiered storage:

- **Current billing expenses in USD**: Stay informed about your current tiered storage expenses.
- **Forecasted month cost in USD**: Estimate your upcoming monthly costs based on current usage.

> - **Data tiered in bytes**: View the volume of data that has been tiered.

Reviewer 2: This will need some changes, but let's wait until the Console PR is approved.

Reviewer 3: ```suggestion
- **Remote tier usage in bytes**: View the volume of data that has been tiered.
```

Current tiered storage configurations
---------------------------------------------

This section provides an overview of the current local cache details and retention policy configurations for tiered storage:

- **Default local retention time (ms)**: Shows the current local data retention set in milliseconds.
- **Default local retention bytes**: Shows the configured volume of data, in bytes, for local retention.

.. _modify-retention-polices:

Modify retention policies 
`````````````````````````````````

1. In the **Tiered storage settings** section, select the ellipsis (three dots) and select **Update tiered storage settings**.
2. Within **Update tiered storage settings** page, adjust the values for:
   
   - Local Cache
   - Default Local Retention Time (ms)
   - Default Local Retention Bytes
  
3. Confirm your adjustments by selecting **Save changes**.

Graphical view of tiered storage costs
------------------------------------------

Gain a visual understanding of your tiered storage expenses:

- **Hourly expense**: Visualize your hourly expenses through graphical representation.

> - **Total cost and forecast**: Gain insights into your overall costs and receive a forecast. This forecast is an estimate based on your most recent usage patterns and trends. It provides an indicative projection and should not be considered as an exact or fixed value.

Reviewer 2: This is hidden in the Console now, let's temporarily remove here as well?

> - **Total cost and forecast**: Gain insights into your overall costs and receive a forecast. This forecast is an estimate based on your most recent usage patterns and trends. It provides an indicative projection and should not be considered as an exact or fixed value.

Reviewer 3: ```suggestion
```

> Detailed storage overview

Reviewer 3: ```suggestion
Remote storage overview
```

-------------------------

> Explore the specifics of your storage usage and configurations:

Reviewer 3: ```suggestion
Explore the specifics of your tiered storage usage and configurations:
```

> - **Used object storage and SSD storage**:Take a deep dive into the storage mediums in use.

Reviewer 2: This will need some changes, but let's wait until the Console PR is approved.

Reviewer 3: ```suggestion
- **Remote storage usage by topics**: Take a deep dive into how much tiered storage is used per topic.
```
